### PR TITLE
SA-1 API

### PR DESCRIPTION
**Pull request by Saksham-1508**
### Changes Made

- Minor bug fixes
- Added API endpoints to fetch data

### CODE REVIEW

**Comment by akshatnema** [comment]
@Saksham-1508 pls correct the PR title format and add some information in it.

File: FusionIIIT/applications/online_cms/migrations/0002_auto_20230418_1520.py
Comment (samay-rgb) [inline FusionIIIT/applications/online_cms/migrations/0002_auto_20230418_1520.py:1]: Remove this file from PR.

**Review by PranshuNayak — CHANGES_REQUESTED** [review]
Remove the migrations of online_cms from this PR

File: FusionIIIT/applications/online_cms/migrations/0002_auto_20230418_1520.py
Comment (GoelYash1) [reply FusionIIIT/applications/online_cms/migrations/0002_auto_20230418_1520.py:1]: Deleted this file

**Review by PranshuNayak — APPROVED** [review]

**Pull request merged**